### ISSUE TITLE
olbase build failing with py 2.7.6

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#4494]
Causing the docker hub image to fail building as well. This is going to be blocking deploys.

```
Hunk #2 succeeded at 307 (offset -1 lines).
Hunk #3 succeeded at 1809 (offset -3 lines).
patching file ./configure
[91mInstalling pip from https://bootstrap.pypa.io/get-pip.py...
[0m[91merror: failed to install pip via get-pip.py
[0m[91m
[0m[91mBUILD FAILED[0m[91m (Ubuntu 16.04 using python-build 20180424)
[0m[91mInspect or clean up the working tree at /tmp/python-build.20210125201447.162
[0m[91mResults logged to /tmp/python-build.20210125201447.162.log
Last 10 log lines:
[0m[91m File "get-pip.py", line 24226, in <module>
main()
File "get-pip.py", line 199, in main
bootstrap(tmpdir=tmpdir)
File "get-pip.py", line 82, in bootstrap
from pip._internal.cli.main import main as pip_entry_point
File "/tmp/tmp_iKCpH/pip.zip/pip/_internal/cli/main.py", line 60
sys.stderr.write(f"ERROR: {exc}")
^
SyntaxError: invalid syntax
[0mThe command '/bin/sh -c pyenv update && pyenv install 2.7.6' returned a non-zero code: 1
build hook failed! (1)
```

**Comment by cclauss** [comment]
pyenv/pyenv#1793

**Comment by cclauss** [comment]
Fixed by us in pyenv/pyenv#1793
```
Downloading Python-2.7.6.tgz...
-> https://www.python.org/ftp/python/2.7.6/Python-2.7.6.tgz
Installing Python-2.7.6...
patching file ./Modules/readline.c
patching file ./Lib/site.py
patching file ./Lib/ssl.py
Hunk #2 succeeded at 430 (offset -5 lines).
patching file ./Modules/_ssl.c
Hunk #2 succeeded at 307 (offset -1 lines).
Hunk #3 succeeded at 1809 (offset -3 lines).
patching file ./configure
Installing pip from https://bootstrap.pypa.io/2.7/get-pip.py...
Installed Python-2.7.6 to /home/openlibrary/.pyenv/versions/2.7.6

Removing intermediate container a2d07e78ed35
 ---> 5a1877077cba
Step 14/40 : RUN pyenv update && pyenv install 3.8.6
```
